doc/script_commands.txt

```diff
@@ -10254,6 +10254,15 @@ in milliseconds. Typically used in item scripts of mercenary scrolls.
 
 ----------------------------------------
 
+*mercenary_delete({<char_id>, <update>})
+
+Remove mercenary of the attached character.
+If <update> is given:
```

Comment:
This seems to imply that if `update` is not given, it won't update the loyalty (but in fact it would increase it by 1). 

How about adding `(defaults to 0 if not given)` to clarify?